osu.Game/Overlays/Settings/Sections/General/UpdateSettings.cs

```diff
@@ -143,8 +143,9 @@ private async Task checkForUpdates()
             }
             finally
             {
-                // This sequence allows the notification to be immediately dismissed.
-                checkingNotification.State = ProgressNotificationState.Cancelled;
+                // This sequence allows the notification to be immediately dismissed without posting a continuation message.
+                checkingNotification.CompletionTarget = null;
+                checkingNotification.State = ProgressNotificationState.Completed;
```

Comment:
This change has significance because setting `Cancelled` also cancels the internal CTS that then cancels the entire rest of the update process in crossfire.

`Completed` doesn't do this but instead posts a completion message. This is resolved by setting `CompletionTarget`.